src/apirequest.ts

```diff
@@ -27,6 +27,7 @@ import {
 } from './api';
 import {isBrowser} from './isbrowser';
 import {SchemaParameters} from './schema';
+import * as h2 from './h2moo';
```

Comment:
what does `moo` stand for; I found the logic otherwise fairly easy to follow, and think I'm just missing an important acronym I should know.

Reply:
It just kinda came out because it rhymed with "2", and I liked the idea of an HTTP cow.

Reply:
😆open to calling it something boring?